drivers/sensor/tmp108/Kconfig

```diff
@@ -1,16 +1,17 @@
 # TMP108 temperature sensor configuration options
 
 # Copyright (c) 2021 Jimmy Johnson <catch22@fastmail.net>
+# Copyright (c) 2022 T-Mobile USA, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
 menuconfig TMP108
 	bool "TMP108 Temperature Sensor"
 	depends on I2C
 	help
-	  Enable driver for the TMP108 temperature sensor.
+	  Enable driver for the TMP108 temperature sensor and/or it's variant
+	  the AMS AS621.
 
 if TMP108
-
```

Comment:
Please don't make unnecessary whitespace changes like this. They clutter up the diff for reviewers and violate the "a commit does one thing" principle. In fact, they are the canonical example of doing two things :).

Slide 14: https://static.sched.com/hosted_files/zephyr2022/a3/HOWTO_Get_Your_Zephyr_Patches_Merged_ZDS_2022.pdf

Reply:
sorry about that, will do